plugins/modules/net_tools/nmcli.py

```diff
@@ -1141,8 +1184,11 @@ def main():
             ip_tunnel_dev=dict(type='str'),
             ip_tunnel_local=dict(type='str'),
             ip_tunnel_remote=dict(type='str'),
+            ssid=dict(type='str'),
+            wifi_sec=dict(type='dict'),
```

Comment:
Since this option contains credentials:
```suggestion
            wifi_sec=dict(type='dict', no_log=True),
```